src/main/java/TLSEmail.java

```diff
@@ -0,0 +1,30 @@
+import java.util.Properties;
+import javax.mail.Authenticator;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+
+public class TLSEmail {
```

Comment:
I think this uses only inside your emailNotificationService. It can be just private method inside your service